webrender/src/platform/macos/font.rs

```diff
@@ -158,9 +165,36 @@ impl FontContext {
         })
     }
 
+    // Assumes the pixels here are linear values from CG
+    fn gamma_correct_pixels(&self, pixels: &mut Vec<u8>, width: usize,
```

Comment:
nit: As far as I understand it, `pixels`'s length cannot change, right? If so, you could probably make it a `&mut [u8]` so you save an indirection.